source/libike/manager.file.cpp

```diff
@@ -405,7 +405,7 @@ bool _CONFIG_MANAGER::file_vpn_del( CONFIG & config )
 
 bool read_line_pcf( FILE * fp, BDATA & name, BDATA & data )
 {
-	char	next;
+	int     next;

```

Comment:
We should not compare unsigned chars to `EOF` (-1), so here the fix is to avoid casting the return value of `fgetc`, otherwise the config will not be loaded correctly (the casted value will be 255 instead of -1)

The exactly same problem happened here: https://bugs.webkit.org/show_bug.cgi?id=144439